src/MadLibs.php

```diff
@@ -0,0 +1,169 @@
+        $helper = $this->getHelper('question');
+
+        $this->printSeparator($input, $output);
+        $question = new Question('<question>Enter 1 if you would like to play again: </question>', 0);
+
+        return $helper->ask($input, $output, $question);
+    }
+
+    private function getSelection(InputInterface $input, OutputInterface $output): int {
+        $helper = $this->getHelper('question');
+
+        $this->printSeparator($input, $output);
+        $question = new Question('Please enter the number for a Mad Lib '
+            . 'from the options above:  ', 0);
+
+        return $helper->ask($input, $output, $question);
+    }
+
+    private function getResponses(InputInterface $input, OutputInterface $output, array $blanks): array {
+        $output->writeln("Fill in the blanks: ");
+        $helper = $this->getHelper('question');
+        $responses = array();
+        foreach ($blanks as $blank) {
+            $question = new Question($blank . ": ", "---");
+            $response = $helper->ask($input, $output, $question);
+            $responses[] = $response;
+        }
+
+        return $responses;
+    }
+
+    private function buildMadLib(array $responses, array $values): string {
+        $result = ""; # final string resulting from putting together blanks and values
+
+        $numValues = count($values);
+        $numResponses = count($responses);
+        $valuesIndex = 0;
+        $responsesIndex = 0;
+
+        while ($valuesIndex < $numValues && $responsesIndex < $numResponses) {
+            $result .= $values[$valuesIndex++];
+            $result .= $responses[$responsesIndex++];
+        }
+        // add remaining items from values and responses arrays to the result string
+        while ($valuesIndex < $numValues) {
+            $result .= $values[$valuesIndex++];
+        }
+        while ($responsesIndex < $numResponses) {
+            $result .= $responses[$responsesIndex++];
+        }
+
+        return $result;
+    }
+
+    private function printSeparator(InputInterface $input, OutputInterface $output) {
+        $output->writeln("=======================================================================");
+    }
+
+    private function printTitleAndOptions(InputInterface $input, OutputInterface $output, array $madLibsOptions) {
+        $output->writeln("\n======================================================================="
+                                . "\n                              MAD  LIBS                                "
+                                . "\n=======================================================================\n");
+
+        $keyIndex = 0;
+        foreach ($madLibsOptions as $optionTitle) {
+            $output->writeln('[' . $keyIndex . '] : <info>' . $optionTitle . '</info>');
+            $keyIndex++;
+        }
+
+        $output->writeln("\n=======================================================================\n");
+    }
+
+}
```

Comment:
GitHub always will point out when there isn't an empty newline at the end of a file. Generally this is a best practice. I'm not 100% sure why but I think it helps with git diffs when things are added to the end of the file